docs/topics/mta-custom-questionnaire.adoc

```diff
@@ -51,7 +51,7 @@ Automated answers are selected based on the tags existing on the application or
 ----
 
 
-Autotagging of applications based on answers::
+Automatic tagging of applications based on answers::
 With this feature, tags are automatically applied to the assessed application or archetype based on the answer if this answer is selected. The tags are transitive. Each tag is defined by the following elements:
```

Comment:
What does "transitive" mean here?

Reply:
@nachandr can you help here ?

Reply:
@mpershina , @sshveta ,  Supposing tags are applied to an application or archetype based on the answers during an assessment, these tags get removed when the assessment is discarded. That's what makes the assessment tags transitive.

Reply:
Added the new suggested wording in the gdoc file, thank you :-)